### DISCUSSION TITLE
Pcm1808 audio amplifier with Micropython?

### DISCUSSION THREAD

**Discussion by LeandroFernandes88** [micropython/micropython#14537] [Hardware & Peripherals]
Hello everybody,

Could someone tell me if there is a library to control Pcm1808 audio amplifier with Micropython?

https://pt.aliexpress.com/item/1005005309444202.html?spm=a2g0o.order_list.order_list_main.5.3b81caa4TVDWPo&gatewayAdapt=glo2bra

If someone know any way to do it and could help with the code, I would appreciate it so much.

**Comment by jonnor** [comment]
I2S output is supported on several ports, and could be used with that board.
